test/emulator_backend.go

```diff
@@ -474,6 +474,20 @@ func (e *EmulatorBackend) DeployContract(
 }
 
 func (e *EmulatorBackend) ReadFile(path string) (string, error) {
+	// These are the scripts/transactions used by the
+	// BlockchainHelpers file. The null character is
+	// used in order to prevent conflicts with user-defined
+	// scripts/transactions.
+	if path == "\x00helper/mint_flow.cdc" {
```

Comment:
Same here: Create a helper function that detects the `"\x00helper/"` prefix and returns the reminder. If the result is non-empty, then switch on just the script names, without the prefix

Reply:
Good idea as well :100: Updated in https://github.com/onflow/cadence-tools/pull/181/commits/e319690e5088c13e135abecc7665c361867dd46d